contrib/roles/linux/docker/tasks/restart_docker.yml

```diff
@@ -6,6 +6,7 @@
   service:
```

Comment:
Can you please comment out the whole `docker.socket` task and add a TODO to follow up on the docker issue you posted.

I tried to deploy on a clean environment and got the following:

```
fatal: [192.168.29.140]: FAILED! => {
    "changed": false,
    "invocation": {
        "module_args": {
            "daemon_reload": false,
            "enabled": null,
            "force": null,
            "masked": null,
            "name": "docker.socket",
            "no_block": false,
            "scope": null,
            "state": "restarted",
            "user": null
        }
    },
    "msg": "Could not find the requested service docker.socket: host"
}

NO MORE HOSTS LEFT *****************************************************************************************************************************************************************************************
        to retry, use: --limit @/home/ubuntu/ovn-kubernetes/contrib/ovn-kubernetes-cluster.retry

root@ubuntu:/home/ubuntu/ovn-kubernetes/contrib# ssh ubuntu@192.168.29.140 cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.4 LTS"
```

Reply:
@lanoxx could you update for @aserdean's comments?  Thanks!

Reply:
This issue should be fixed in 18.09.1 which will probably be released next week as the rc1 was out today. I would suggest to wait a few more days to confirm its fixed upstream and then just merge it as is.

Reply:
Here is the link to the releases:
https://github.com/docker/docker-ce/releases

Reply:
Thanks for the update @lanoxx. I will keep an eye on it and retry it once 18.09.1 is out.

Reply:
@lanoxx 18.09.1 is still not out. Do you mind splitting the PR into two?
I can merge https://github.com/openvswitch/ovn-kubernetes/pull/506/commits/e1a15252ba0cb989a87214ff658ea2e16f0a7f65 as is, and wait for 18.04 support until 18.09.1 is out.

Reply:
@lanoxx as pointed by @ionutbalutoiu can you modify the rest of the files where `fetch` is used and split the PR in individual commits?